src/modals/LinkProjectModal.js

```diff
@@ -38,16 +59,49 @@ const LinkProjectModal = ({linkProject, onCloseDialog, study, allProjects}) => {
       onSubmit={onSubmit}
     >
       {formikProps => (
-        <Modal
-          as={Form}
-          onSubmit={formikProps.handleSubmit}
-          open={true}
-          onClose={onCloseDialog}
-          closeIcon
-        >
+        <Modal open={true} onClose={onCloseDialog} closeIcon>
           <Modal.Header content="Link an Existing Analysis Project" />
-          <Modal.Content>
-            <p>Link a Cavatica Project to this study</p>
+          <Modal.Content className="pb-0">
+            <Header
+              as="h3"
+              floated="left"
+              content="Link a Cavatica Project to this study"
+            />
+            <Button
+              basic
+              primary
+              type="button"
+              floated="right"
+              size="mini"
+              icon="sync"
+              loading={syncing}
+              content="SYNC PROJECT"
+              onClick={sync}
+            />
+            {syncing ? (
+              <Message
+                icon={<Icon name="sync" loading />}
+                header="Syncing Cavatica Projects"
+                content="This could take a moment..."
+              />
+            ) : (
+              <Message
+                warning
+                icon="attention"
+                header="Attention"
+                content="If you created the project from Cavatica, plese sync it before linking"
```

Comment:
Maybe better phrased as "Projects created recently from Cavatica may require a sync before being available for linking."